.drone.yml

```diff
@@ -93,6 +93,17 @@ pipeline:
       matrix:
         TEST_SUITE: phpunit
 
+  js-tests:
+    image: owncloudci/php:${PHP_VERSION}
+    pull: true
+    commands:
+      - cd /var/www/owncloud/apps/user_management/tests/js
```

Comment:
makefile would be nicer

Reply:
any hint where this is done properly?

Reply:
here but I don't expect that complex a Makefile for starters: https://github.com/owncloud/customgroups

let's use the way how core does it: https://github.com/owncloud/core/blob/master/Makefile#L171

also see http://www.olioapps.com/blog/the-lost-art-of-the-makefile/ if you want to make it a little bit for fancy ;-)